identity/frontend/package.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
❌ Let's remove the `frontend` folder for now. It only contains (here) unused commands/dependencies. I would create the frontend structure at a later stage